basic/datadog_log_destination/datadog_log_destination.tf

```diff
@@ -0,0 +1,53 @@
+############ Creating a Kinesis Firehose Delivery Stream to Datadog ##############
+
+# This module creates a AWS Kinesis Firehose Delivery Stream that pipes logs to Datadog
+module "datadog_connector" {
+  source  = "symopsio/datadog-connector/aws"
+  version = ">= 2.0.0"
+
+  environment = "main"
+
+  # This variable should NOT be checked into version control!
+  # Set it in an untracked tfvars file (e.g. `secrets.tfvars`)
+  # or as an environment variable: `export TF_VAR_datadog_access_key="my-access-key"`
+  datadog_access_key = var.datadog_access_key
+}
+
+resource "sym_log_destination" "datadog" {
+  type = "kinesis_firehose"
+
+  # The Runtime Permission Context has Kinesis Firehose permissions from the aws/kinesis-firehose add-on
+  integration_id = sym_integration.runtime_context.id
+
+  settings = {
+    # The firehose stream name is outputted by the datadog_connector module
+    stream_name = module.datadog_connector.firehose_name
+  }
+}
+
+
+resource "sym_flow" "this" {
```

Comment:
Wondering if the flow should be in a different file from the log destination 🤷‍♀️ but maybe that's too much